lib/opensusebasetest.pm

```diff
@@ -231,17 +231,19 @@ done", "binaries-with-missing-libraries.txt", {timeout => 60, noupload => 1});
 
 =head2 investigate_yast2_failure
 
- investigate_yast2_failure();
+ investigate_yast2_failure(logs_path => $logs_path);
 
-Inspect the YaST2 logfile checking for known issues.
+Inspect the YaST2 logfile checking for known issues. logs_path can be a directory where logs are saved
+e.g. /tmp. In that case the function will parse /tmp/var/log/YaST2/y2logs* files.
 
 =cut
 sub investigate_yast2_failure {
-    my ($self) = shift;
-
+    my ($self, %args) = @_;
+    my $logs_path = $args{logs_path} . '/var/log/YaST2';
```

Comment:
I would suggest here to remove the trailing forward slash (/) in case the user would pass it in that way, but from your comment is clear what to expect.

Reply:
If the user passes "/" at the end of the logs_path, it will still work. in the opposite scenario, it won't work.

Reply:
how my bash! I didn't remember that two slashes in a path is entirely harmless so linux scripts ignore them. Thanks for the reminder.